drivers/lora/sx126x.c

```diff
@@ -381,34 +381,13 @@ static int sx126x_lora_init(struct device *dev)
 
 	LOG_DBG("Initializing %s", DT_INST_LABEL(0));
 
-	dev_data.reset = device_get_binding(GPIO_RESET_LABEL);
-	if (!dev_data.reset) {
-		LOG_ERR("Cannot get pointer to %s device", GPIO_RESET_LABEL);
+	if (sx12xx_configure_pin(reset, GPIO_OUTPUT_ACTIVE) ||
```

Comment:
Macros no longer used (to be removed):
- GPIO_RESET_LABEL
- GPIO_RESET_FLAGS
- GPIO_BUSY_LABEL
- GPIO_BUSY_FLAGS
- GPIO_DIO1_LABEL
- GPIO_DIO1_FLAGS
- GPIO_ANTENNA_ENABLE_LABEL
- GPIO_ANTENNA_ENABLE_FLAGS

Reply:
Thanks! I thought I did that already. I have removed these from the updated commit.